test/model/property_test.gd

```diff
@@ -10,109 +10,124 @@ const __source = "res://addons/pandora/model/property.gd"
 
 
 func test_string_property() -> void:
-	var property = PandoraProperty.new("123", "property", "string", "Hello World")
-	var new_property = PandoraProperty.new("", "", "", "")
+	var property = PandoraProperty.new("123", "property", "string")
+	property.set_default_value("Hello World")
+	var new_property = PandoraProperty.new("", "", "")
 	new_property.load_data(property.save_data())
 	assert_that(new_property).is_equal(property)
 	
 	
 func test_string_property_wrong_type() -> void:
-	var property = PandoraProperty.new("123", "property", "reference", "Hello World")
-	var new_property = PandoraProperty.new("", "", "", "")
+	var property = PandoraProperty.new("123", "property", "reference")
+	property.set_default_value("Hello World")
+	var new_property = PandoraProperty.new("", "", "")
 	new_property.load_data(property.save_data())
 	assert_that(new_property.get_default_value()).is_null()
 	
 	
 func test_int_property() -> void:
-	var property = PandoraProperty.new("123", "property", "int", 123)
-	var new_property = PandoraProperty.new("", "", "", "")
+	var property = PandoraProperty.new("123", "property", "int")
+	property.set_default_value(123)
+	var new_property = PandoraProperty.new("", "", "")
 	new_property.load_data(property.save_data())
 	assert_that(new_property).is_equal(property)
 	
 	
 func test_int_property_wrong_type() -> void:
-	var property = PandoraProperty.new("123", "property", "string", 123)
-	var new_property = PandoraProperty.new("", "", "", "")
+	var property = PandoraProperty.new("123", "property", "string")
+	property.set_default_value(123)
+	var new_property = PandoraProperty.new("", "", "")
 	new_property.load_data(property.save_data())
-	assert_that(new_property.get_default_value()).is_null()
+	assert_that(new_property.get_default_value()).is_equal("")
```

Comment:
These tests now change as it is impossible to create a `PandoraProperty` that has a valid type but an invalid default value.